tests/CMakeLists.txt

```diff
@@ -11,6 +11,20 @@ if (NOT AUTOPAS_BUILD_TESTS)
     return()
 endif ()
 
+option(AUTOPAS_ENABLE_COVERAGE "Enable code coverage, requires GCC and CMAKE_BUILD_TYPE=Debug." OFF)
+message(STATUS "Enable code coverage: " ${AUTOPAS_ENABLE_COVERAGE})
```

Comment:
for consistency maybe change the messages so that it either says
`LCOV Coverage report enabled`
or
`LCOV Coverage report disabled`
This is fine to do with two calls to `message`

Reply:
done in [889f266](https://github.com/AutoPas/AutoPas/pull/888/commits/889f26618f64e9d66805dd320b2ef8d07f14578a)